src/components/AddIncomeModal/AddIncomeModal.jsx

```diff
@@ -0,0 +1,125 @@
+import { useForm } from 'react-hook-form';
+import {
+  Button,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightAddon,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  NumberInput,
+  NumberInputField,
+  Select,
+  Textarea
+} from '@chakra-ui/react';
+import i18next from 'i18next';
+import moment from 'moment';
+
+export const AddIncomeModal = ({ isOpen, onClose, onSubmit }) => {
+  const { register, handleSubmit } = useForm({
+    defaultValues: {
+      wallet: 'wallet-1-default',
+      date: moment(new Date()).format('YYYY-MM-DD'),
```

Comment:
What's the alternative?

Reply:
In theory, we can handle dates from Chakra's Input, but I don't know if it will be a good way of handling dates in our case. I mean, I don't have that experience)
https://chakra-ui.com/docs/components/input#input-methods-other-than-text

Reply:
Anton suggests to use dayjs instead (because moment doesn't have support already) or use just object Date (without additional adding the library). I tried to make some variants, and there are some solutions (the second one is better):
<img width="639" alt="Снимок экрана 2022-07-22 в 11 29 05 AM" src="https://user-images.githubusercontent.com/47990337/180415110-28bcb1cd-071b-4727-9617-a31d9220311c.png">
<img width="639" alt="Снимок экрана 2022-07-22 в 11 39 54 AM" src="https://user-images.githubusercontent.com/47990337/180415119-b7b178d1-68bf-4a68-901b-9a0454cf3dc8.png">
<img width="639" alt="Снимок экрана 2022-07-22 в 11 47 54 AM" src="https://user-images.githubusercontent.com/47990337/180415123-ab85ed8c-83b6-495a-9d21-ac517653ab0e.png">

Reply:
I only worked with momentjs and it was good for converting formats for BE, since we don't know how they are going to be accepting date objects

Reply:
I also think you should use '.' as a separator. Using native Date for now is okay

Reply:
@itsnotpolza yeah moment.js maintainers even suggested looking for something else on their home page.
and about BE we actually can assume they will use some datatype, and Date object allows us to do anything about it.

so I am fine with libraries in general, but just wondering if we need them just to one-row operation
and it's not good practise to use some libraries in projects which not actively maintained